src/interfaces/IStandardGovernor.sol

```diff
@@ -157,7 +158,8 @@ interface IStandardGovernor is IBatchGovernor {
             ProposalState state,
             uint256 noVotes,
             uint256 yesVotes,
-            address proposer
+            address proposer,
+            uint256 quorum
```

Comment:
is quorum even needed here? no quorum, no need to return parameter.

Reply:
As it stands, `getProposal` is the only way for any font end to query the current tally of yes and no votes, so this function is useful to get the entire state, and to be generic, would need to return the quorum as well.